builds/msvc/vs2013/libbitcoin-explorer/libbitcoin-explorer.vcxproj

```diff
@@ -79,6 +79,7 @@
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer.hpp" />
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer\impl\prop_tree.ipp" />
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer\impl\utility.ipp" />
+    <ClInclude Include="..\..\..\..\include\bitcoin\explorer\async_client.hpp" />
```

Comment:
`async_client` was removed in a4e64190f4afebfd70d8cc90efaf8020169b397b, so this appears to be a merge issue.

Also, the new command files should be moved from the `stub` subfolder. The `model/generate.xml` file includes the attribute `stub="true"`, which should be removed. That will cause the generated header to be produced in the correct location. The old header should be deleted and the .cpp file should be manually moved.
```xml
  <command symbol="qrcode" category="WALLET" output="raw" stub="true" description=...>
```